src/lib/kw_config_loader.sh

```diff
@@ -20,24 +20,38 @@ TARGET="$VM_TARGET"
 
```

Comment:
Just to make sure: it seems to me that the changes in this file are only for `kw config`, in the sense that these changes should be imperceptible to other points in the project (i.e., places where there are `load_<feature>_config` calls). Is this correct?

Configurations are fundamental to the proper work of kw, so we need to be extra careful here. From my review I'm not sure if these changes have side effects elsewhere, so see if you can double check this matter, please.

Reply:
yes, that is correct. Changes do not alter current behavior, only extend it. It is a backwards compatible change: does not break existing code.